src/components/styles.js

```diff
@@ -0,0 +1,130 @@
+import styled, { css, createGlobalStyle } from "styled-components";
```

Comment:
split these styles to globalStyles and component level styles and inherit where needed.